src/app/login/login.component.ts

```diff
@@ -117,7 +121,11 @@ export class LoginComponent implements OnInit {
         (res: any) => {
           if (res.statusCode === 200) {
             if (res.data.previlegeObj && res.data.previlegeObj[0]) {
-              localStorage.setItem(
+              this.cookieService.set('Jwttoken', res.data.Jwttoken);
+              delete res.data.Jwttoken;
+              console.log('>>>>login', JSON.stringify(res.data));
+
+              this.sessionstorage.setItem(
```

Comment:
> _⚠️ Potential issue_
> 
> **Remove console.log of sensitive login data**
> 
> The console.log statement exposes sensitive login response data. This could lead to security vulnerabilities in production.
> 
> ```diff
> -  console.log('>>>>login', JSON.stringify(res.data));
> ```
> 
> 📝 Committable suggestion
> 🧰 Tools

console removed